python/minsim.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" ?>
```

Comment:
shared/sim_test_config.xml?

Reply:
Probably before we review this it should work and the dependency PRs should be OK :/

Reply:
I was reading this for other reasons, would be silly to not post my comments and have to start again another time. ;)

Reply:
minimum-sim-config.xml is what I went with, since I now also use this config manually.